src/main/java/com/minecolonies/entity/pathfinding/PathNavigate.java

```diff
@@ -238,21 +258,21 @@ else if (entity.isInWater())
 
                 this.getPath().setCurrentPathIndex(oldIndex);
 
-                Vec3 vec3 = this.getPath().getPosition(this.entity);
+                Vec3d Vec3d = this.getPath().getPosition(this.entity);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)